Governance/tech/massdata.md

```diff
@@ -66,7 +66,7 @@ Please note mdss sub-commands work only interactively or on the `copyq` queue. T
 
 ### Monitoring MDSS usage
 
-Unfortunately, there is no command to check the usage by user-id as for /g/data and /scratch. The only way to get this information currently is to ask help<at>nci.org.au. The NCI administrators can access this information for any CI of the group.
+Unfortunately, there is no command to check the usage by user-id as for /g/data and /scratch. The only way to get this information currently is to ask help@nci.org.au. The NCI administrators can access this information for any CI of the group.
```

Comment:
I suspect `<at>` was used to avoid bot spamming, but maybe that's up to NCI to filter!